src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate+QuickActions.swift

```diff
@@ -0,0 +1,96 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+// MARK: - Quick Actions
+
+extension AppDelegate {
+
+	/// General identifier for the 'add diary entry' shortcut action
+	private static let shortcutIdDiaryNewEntry = "de.rki.coronawarnapp.shortcut.diarynewentry"
+
+	func application(_ application: UIApplication, performActionFor shortcutItem: UIApplicationShortcutItem, completionHandler: @escaping (Bool) -> Void) {
+		handleShortcutItem(shortcutItem)
+	}
+
+	/// Checks wether the application is launched from a shortcut or not.
+	///
+	/// - Parameter launchOptions: the launch options passed in the app launch (will/did) functions
+	/// - Returns: `false` if the application was launched(!) from a shortcut to prevent further calls to `application(_:performActionFor:completionHandler:)`
+	func handleQuickActions(with launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
About the timing: it's the ENFramework which needs to be started first, before the rest of the UI bootstraps (which is the real flaw that that might be reviewed). I could listen for changes in the `ENAExposureManagerObserver` to get rid of the timing but basically I don't need to care about the ENF state, so I just wait 'long enough' to have some UI I can go on with.

Reply:
I think this timing is really fragile and 0.2 is a magic number.
It can fail for many reasons.
Maybe older devices, which needs longer.
Maybe apple changing something internally that takes longer.
Maybe we will add something to showHome, which takes longer.
In my opinion this is fragile.
Would appreciate if the coordinator handles it, after the view was really bootstrapped.

Reply:
I'm waiting until a UI is available which the app can apply the shortcut action. The whole bootstrap process is too complex to 'just refactor it' on the go.

Reply:
Nice! Thanks! 🚀